exercises/flatten-array/FlattenArrayExample.swift

```diff
@@ -0,0 +1,37 @@
+import Foundation
+
+func flattenArray(list: [Any?]) -> [Any] {
+
+    var anyArray = [Any]()
+
+    func extractArrayElements(array: [Any?]) {
+
+        for element in array {
+
+            if let j = element {
+
+                let anyObjectArray = j as? [Any?]
+
+                if let forceUnwrappedArray = anyObjectArray {
```

Comment:
This name here is misleading. force unwrapping only happens when we use the ! operator.